### DISCUSSION TITLE
Issue with installation on Ubuntu 18.04

### DISCUSSION THREAD

**Discussion by TitouanTyack** [USC-ACTLab/crazyswarm#436] [Q&A] [answered]
Hello,

I wanted to test crazyswarm on a fresh Ubuntu 18.04 install.
I followed the "Physical Robots and Simulation" docs : https://crazyswarm.readthedocs.io/en/latest/installation.html
When I try to install the dependencies, it returns me this error :
![Screenshot from 2021-07-23 16-35-48](https://user-images.githubusercontent.com/85453067/126798466-ef784993-ae3b-471e-94d0-dcdfcd795b2b.png)

Do you have any idea of what can I do ?

Thanks for you help

**Comment by whoenig** [comment, marked as answer]
Use `export CSW_PYTHON=python` rather than `export CSW_PYTHON=python2`.

**Reply by TitouanTyack** [reply]
It worked, thanks a lot
